clang/tools/3c/3CStandalone.cpp

```diff
@@ -75,15 +75,19 @@ are "#include"-d by that file and it is an error if any other file changes.
 // Skip the 2 initial newlines.
 static cl::extrahelp MoreHelp(MoreHelpStr + 2);
 
-static cl::opt<bool> OptDumpIntermediate("dump-intermediate",
-                                         cl::desc("Dump intermediate "
-                                                  "information"),
-                                         cl::init(false), cl::cat(_3CCategory));
+// Letting clang-format reflow this declaration gives very inconsistent
```

Comment:
I think manually formatting the option declarations is a worthwhile tradeoff. Alternatively, I'd be happy if someone knows an incantation to make clang-format handle these consistently.